left_column.md

```diff
@@ -100,10 +100,10 @@ The schedule is subject to change.
 **25 May (M)**
 - 9:00 - 9:30
   [Welcome and practical information](https://github.com/coderefinery/workshop-intro/blob/master/README.md)
-  (TBA)
+  (Radovan)
 - 9:30 - 12:00
   [Introduction to version control - part 1/2](https://coderefinery.github.io/git-intro/)
-  (TBA)
+  (Radovan/Stefan)
```

Comment:
I wanted to do motivation (episode 1) part but I can do more.

Reply:
I can do something too.

Reply:
I can do everything after motivation or parts of it. either way reserved the first half of day for this.